modules/weblog/metrics.go

```diff
@@ -131,6 +141,21 @@ func newReqCustomField(fields []customField) map[string]metrics.CounterVec {
 	return cf
 }
 
+func newReqCustomTimeField(fields []customTimeField) map[string]*customTimeFieldMetrics {
+	cf := make(map[string]*customTimeFieldMetrics)
+	for _, f := range fields {
+		var buckets []float64
+		for _, value := range f.Histogram {
+			buckets = append(buckets, value*1e6)
+		}
+		cf[f.Name] = &customTimeFieldMetrics{
+			Time:     newWebLogSummary(),
+			TimeHist: metrics.NewHistogram(buckets),
```

Comment:
Do we really need default bucket if histogram is not defined? I see you add histogram chart if len(f.Histogram) > 0, seems the answer is no